linux_pipeline/Jenkinsfile_developer_patch_validation

```diff
@@ -9,8 +9,8 @@ pipeline {
     parameters {
         choice(choices: 'git://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git\nhttps://git.kernel.org/pub/scm/linux/kernel/git/davem/net-next.git\nazure_kernel', description: 'Kernel tree repo', name: 'KERNEL_GIT_URL')
         string(defaultValue: "master", description: 'Branch to be built', name: 'KERNEL_GIT_BRANCH')
```

Comment:
For commit message:

[developer] Added hostname option for scp patches and improved params descriptions